src/applications/mhv/shared/hooks/useDatadogRum.jsx

```diff
@@ -20,12 +20,17 @@ const initializeDatadogRum = config => {
   }
 };
 
-const useDatadogRum = config => {
+const useDatadogRum = (config, userInfo) => {
   useEffect(
     () => {
       initializeDatadogRum(config);
+      if (userInfo.loggedIn && environment.isStaging()) {
```

Comment:
NOTE: if this works in, staging. We can explore adding a feature flag vs just turning it on.